config/webpack.dev.js

```diff
@@ -3,6 +3,8 @@ const common = require("./webpack.common.js");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const fs = require("fs");
 
+const htmlLayout = fs.readFileSync("./site/layout.html");
```

Comment:
optional: Move this into the exported function (after line 8) so it only runs when invoked.

In this circumstance it changes basically nothing, but its generally good form to avoid side effects on import when possible.